### ISSUE TITLE
wasm

### ISSUE THREAD

**Issue by gedw99** [qri-io/skytf#6]
why not use wasm for the transformation code that needs to run ?
that means you can write them in go, c, etc etc and they are sandboxed.

just see everyone reinvent this everywhere. same for blockchain contracts like solidity. just use WASM.... :)

rant i know... but seems logical to me

**Comment by b5** [comment]
OooooOoh interesting! Our main reason for starting with skylark is data scientists are far more comfortable with python-style syntax, and capacity for parallel execution . Totally agreed that reinventing the wheel is irritating, and we only went this way b/c learning another language for data science is a bit of a non-starter for the people we've talked to.

We do, however, have support for multiple syntaxes built elsewhere, so there's nothing stopping us from writing a WASM transform implementation as well 😄.  I'd love to live in a world where we can use WASM for transforms.
